rasa/utils/io.py

```diff
@@ -277,3 +277,21 @@ def validate(self, document: Document) -> None:
                 raise ValidationError(message=error_message)
 
     return ExportPathValidator
+
+
+def questionary_not_empty_text_validator(error_message: Text) -> Type["Validator"]:
```

Comment:
can we refactor the two validators to use some common behavior?

I thought of sth like:

```python
def path_validator(some_params, error_message) -> Validator:
  def is_valid(text: Text) -> bool:
     return text.endswith("yml")
  
  return create_validator(is_valid, error_message)

def blank_validator(some_params, error_message) -> Validator:
  def is_valid(text: Text) -> bool:
     return text != ""
  
  return create_validator(is_valid, error_message)

def create_validator(f, message) -> Type[Validator]:
  # Create class and use given function `f` to validate things

Reply:
haha, i think this is basically rewriting the from_callable isn't it?

Reply:
probably :-D we can remove the shizzle as soon as tom merges my pr :-D

Reply:
hm maybe this doesn't need to be done now then?

Reply:
mhm, but who knows whether we remember to this once the pr is merged and then it's stuck in the code forever.

Reply:
hahaha ok true